src/style-anchorlink.scss

```diff
@@ -28,26 +30,17 @@
     padding: var(--ds-size-100, $ds-size-100) var(--ds-size-200, $ds-size-200) var(--ds-size-100, $ds-size-100) var(--ds-size-200, $ds-size-200);
 
     &:focus {
-      background-color: unset;
       outline: unset;
     }
   }
-}
 
-:host([active]),
-:host(:hover) {
-  a {
-    color: var(--ds-color-text-primary-default, $ds-color-text-primary-default) !important;
+  .hyperlink {
+    white-space: unset;
```

Comment:
What was the reason we needed to add this?

Reply:
This was done because the auro-hyperlink were causing for text not to wrap in anchornav

Reply:
Just add a comment saying as much in the CSS. We aren't going to remember that in 6 months.